presentation-compiler/test/dotty/tools/pc/tests/completion/CompletionSnippetSuite.scala

```diff
@@ -289,7 +289,8 @@ class CompletionSnippetSuite extends BaseCompletionSuite:
           |}
           |""".stripMargin,
       "scala.util.Try@@(1)",
-      "scala.util.Try(1)"
+      "scala.util.Try(1)",
+      assertSingleItem = false
```

Comment:
Why do we need the change here, what do we get additionally?

Reply:
It will now both offer `Try()` as apply (insert text will be `Try`) and `Try` as module (insert text also `Try`).